src/utils/generate-type-from-schema.test.ts

```diff
@@ -62,33 +63,47 @@ describe('generateTypeFromSchema()', () => {
     jest.clearAllMocks()
   })
 
+  it('immediately returns when types=false', async () => {
```

Comment:
nit: either spell out the sentence or use real programming syntax `when types === false`

Reply:
this test is removed. But paid attention to that in the added tests to `load.test.ts`